components/mm-components/progress-indicator/MmProgressBar.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div role="progressbar" aria-valuemin="0" :aria-valuemax="maxValue" :aria-valuenow="normalizedValue"
+        class="progress-bar" :style="{ height: height + 'px' }">
+        <div class="progress-bar-background" :style="backgroundStyle">
+        </div>
+        <div class="progress-bar-active" :style="{ width: activeValue + '%' }">
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        value: {
+            type: [Number, String],
+            default: 0,
+        },
+        maxValue: {
+            type: [Number, String],
+            default: 100,
+        },
+        height: {
+            type: [Number, String],
+            default: 4,
+        }
```

Comment:
There should be an additional prop to adjust the width of the progress bar. I see it has a 100% width on it but if it is to be made smaller it will require a wrapper div to change its width. If there are multiple small ones in line then each will need a wrapper div to contain their size. I think this component should be responsible for its width.